Google.PowerShell.DotnetCore/project.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Google.PowerShell",
+  "title": "Google Cloud Tools for PowerShell",
+  "description": "Google Cloud Tools for PowerShell, targeting the .NET Core runtime",
+  "copyright": "Google Inc. 2016",
+  "version": "1.0.0",
+
+  "runtimes": {
+    "win81-x64": {},
+    "win10-x64": {},
+    "ubuntu.16.04-x64": {},
+    "osx.10.11-x64": {}
+  },
+
+  "configurations": {
+    "Linux": {
+      "buildOptions": {
+        "define": [ "UNIX" ]
+      }
+    }
+  },
+
+  "buildOptions": {
+    "define": [ "CORECLR" ],
+    "compile": {
+      "include": [
+        "../Google.PowerShell/Common/*.cs",
+        "../Google.PowerShell/Compute/*.cs",
+        "../Google.PowerShell/Dns/*.cs",
+        "../Google.PowerShell/Properties/*.cs",
+        "../Google.PowerShell/Provider/*.cs",
+        "../Google.PowerShell/Sql/*.cs",
+        "../Google.PowerShell/Storage/*.cs"
+      ]
+    }
+  },
+
+  "frameworks": {
+    "netstandard1.6": {
+      "imports": "dnxcore50",
+      "buildOptions": {
+        "define": [ "CORECLR" ]
+      },
+      "dependencies": {
+        "System.Management.Automation": "1.0.0-*",
+        "Portable.BouncyCastle": "1.8.1.1",
```

Comment:
I don't believe we depend on this directly, but rather bring it in through the Google client library infrastructure. So it might be OK to remove it.